modules/cnf-programming-the-sriov-operator.adoc

```diff
@@ -0,0 +1,398 @@
++
+.Example output
+[source,terminal]
+----
+NAME                                            READY       STATUS      RESTARTS    AGE
+fpga-driver-daemonset-8xz4c                     1/1         Running     0           15d
+fpgainfo-exporter-vhvdq                         1/1         Running     1           15d
+N3000-controller-manager-b68475c76-gcc6v        2/2         Running     1           15d
+N3000-daemonset-5k55l                           1/1         Running     1           15d
+N3000-discovery-blmjl                           1/1         Running     1           15d
+N3000-discovery-lblh7                           1/1         Running     1           15d
+sriov-device-plugin-j5jlv                       1/1         Running     1           15d
+sriov-fec-controller-manager-85b6b8f4d4-gd2qg   1/1         Running     1           15d
+sriov-fec-daemonset-kqqs6                       1/1         Running     1           15d
+----
+The following section provides information on the installed pods:
+* `fpga-driver-daemonset` provides and loads the required Open Programmable Accelerator Engine (OPAE) drivers
+* `fpgainfo-exporter` provides N3000 telemetry data for Prometheus
+* `N3000-controller-manager` applies N3000Node CRs to the cluster and manages all the operand containers
+* `N3000-daemonset` is the main worker application
+* `N3000-discovery` discovers N3000 Accelerator devices installed and labels worker nodes if devices are present
+* `sriov-device-plugin` expose the FEC virtual functions as resources under the node
+* `sriov-fec-controller-manager` applies CR to the node and maintains the operands containers
+* `sriov-fec-daemonset` is the main worker application
++
+[NOTE]
+====
+This output shows the `N3000` and the `sriov-fec` FEC pods.
+====
+
+
+. Retrieve all the nodes containing one of the supported vRAN FEC accelerator devices:
++
+[source,terminal]
+----
+$ oc get sriovfecnodeconfig
+----
++
+.Example output
+[source,terminal]
+----
+NAME             CONFIGURED
+node1            Succeeded
+----
+
+. Find the physical function (PF) of the SR-IOV FEC accelerator device to be configured:
+
++
+[source,terminal]
+----
+$ oc get sriovfecnodeconfig node1 -o yaml
+----
++
+.Example output
+[source,yaml]
+----
+status:
+    conditions:
+    - lastTransitionTime: "2021-03-19T17:19:37Z"
+      message: Configured successfully
+      observedGeneration: 1
+      reason: ConfigurationSucceeded
+      status: "True"
+      type: Configured
+    inventory:
+        sriovAccelerators:
+        - deviceID: 0d5c
+          driver: ""
+          maxVirtualFunctions: 16
+          pciAddress: 0000.1d.00.0 <1>
+          vendorID: "8086"
+          virtualFunctions: [] <2>
+----
+<1> This field indicates the PCI Address of the card.
+<2> This field shows that the virtual functions are empty.
+
+.  Configure the FEC device with the desired setting.
+
+.. Create the following custom resource (CR) and save the YAML in the `sriovfec_n3000_cr.yaml` file.
```

Comment:
Ideally, the instructional text would follow the pattern that you established in the first file.

End the instructional text with a colon.